app/build.gradle

```diff
@@ -15,6 +15,7 @@ android {
         versionCode 1
         versionName "1.0"
         buildConfigField("String", "API_BASE_PATH", apiBasePath)
```

Comment:
this is going to be added in the github/local config as well

Reply:
config used in gradle.properties

Reply:
yes, and this will be part of config used github action